onnx/backend/test/case/node/image_decoder.py

```diff
@@ -40,6 +39,13 @@ def generate_checkerboard(width, height, square_size):
 
 
 def generate_test_data(extension, pixel_format="RGB", h=40, w=40, tile_sz=5):
+    try:
+        # pylint: disable=import-outside-toplevel
+        import cv2
+    except ImportError as e:
+        raise ImportError(
+            "opencv-python must be installed to use the reference implementation of the ImageDecoder operator"
```

Comment:
Perhaps something like 
```suggestion
            "opencv-python must be installed to generate test data for the ImageDecoder operator"
```

Reply:
Good catch. Updated. Thanks!